test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetTrustTests.cs

```diff
@@ -0,0 +1,564 @@
+                            <author name=""signer"">
+                                <certificate fingerprint=""abcdef"" hashAlgorithm=""SHA256"" allowUntrustedRoot=""false"" />
+                            </author>
+                        </trustedSigners>
+                    </configuration>";
+
+                var expectedAuthorContent = $@"Registered trusted signers:
+                     1.   signer [author]
+                          Certificate fingerprint(s):
+                            SHA256 - abcdef
+                    ";
+                File.WriteAllText(nugetConfigPath, nugetConfigContent);
+
+                //Act
+                CommandRunnerResult result = _msbuildFixture.RunDotnet(
+                    pathContext.WorkingDirectory,
+                    $"nuget trust list --configfile {nugetConfigPath}");
+
+                // Assert
+                result.Success.Should().BeTrue();
+                SettingsTestUtils.RemoveWhitespace(result.Output).Should().Contain(SettingsTestUtils.RemoveWhitespace(expectedAuthorContent));
+            }
+        }
+
+        [Fact]
+        public void DotnetTrust_ListAction_WithRelativePathNugetConfig_Succeeds()
+        {
+            using (SimpleTestPathContext pathContext = _msbuildFixture.CreateSimpleTestPathContext())
+            {
+                // Arrange
+                var nugetConfigFileName = "NuGet.Config";
+                var nugetConfigContent = $@"<?xml version=""1.0"" encoding=""utf-8""?>
+                    <configuration>
+                        <trustedSigners>
+                            <author name=""signer"">
+                                <certificate fingerprint=""abcdef"" hashAlgorithm=""SHA256"" allowUntrustedRoot=""false"" />
+                            </author>
+                        </trustedSigners>
+                    </configuration>";
+
+                var expectedAuthorContent = $@"Registered trusted signers:
+                     1.   signer [author]
+                          Certificate fingerprint(s):
+                            SHA256 - abcdef
+                    ";
+                var nugetConfigPath = Path.Combine(pathContext.WorkingDirectory, nugetConfigFileName);
+                File.WriteAllText(nugetConfigPath, nugetConfigContent);
+
+                //Act
+                CommandRunnerResult result = _msbuildFixture.RunDotnet(
+                    pathContext.SolutionRoot,
+                    $"nuget trust list --configfile ..{Path.DirectorySeparatorChar}{nugetConfigFileName}");
+
+                // Assert
+                result.Success.Should().BeTrue();
+                SettingsTestUtils.RemoveWhitespace(result.Output).Should().Contain(SettingsTestUtils.RemoveWhitespace(expectedAuthorContent));
+            }
+        }
+
+        [CIOnlyTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task DotnetTrust_AuthorAction_RelativePathConfileFile_Succeeds(bool allowUntrustedRoot)
```

Comment:
nit : test case name `DotnetTrust_AuthorAction_RelativePathConfigFile_Succeeds`

Reply:
Ok. I'll address later in another PR.